lisp/ein-notebook.el

```diff
@@ -1584,10 +1584,12 @@ This hook is run regardless the actual major mode used."
                         (auto-complete-mode +1))
     (ein:use-ac-jedi-backend (ein:jedi-complete-on-dot-install ein:notebook-mode-map)
                              (auto-complete-mode +1))
-    (ein:use-company-backend (add-to-list 'company-backends 'ein:company-backend)
-                             (company-mode +1))
+    (ein:use-company-backend 
```

Comment:
Not really necessary as I would not expect a user to set `ein:completion-backend` to use the company or company-jedi (to be implemented someday!) and not have company installed. In fact, having emacs raise an error here might be useful as it would tell someone that they had forgetten to install or load company.